src/Umbraco.Web.UI.Client/src/common/directives/components/editor/umbeditorcontentheader.directive.js

```diff
@@ -15,36 +15,30 @@
             }
 
             scope.isNew = scope.content.state == "NotCreated";
-            
+
             localizationService.localizeMany([
-                scope.isNew ? "placeholders_a11yCreateItem" : "placeholders_a11yEdit",
-                "placeholders_a11yName",
-                scope.isNew ? "general_new" : "general_edit"]
+                    scope.isNew ? "placeholders_a11yCreateItem" : "placeholders_a11yEdit",
+                    "placeholders_a11yName"]
             ).then(function (data) {
-
                 scope.a11yMessage = data[0];
                 scope.a11yName = data[1];
-                var title = data[2] + ": ";
                 if (!scope.isNew) {
                     scope.a11yMessage += " " + scope.content.name;
-                    title += scope.content.name;
+
                 } else {
                     var name = editorState.current.contentTypeName;
                     scope.a11yMessage += " " + name;
                     scope.a11yName = name + " " + scope.a11yName;
-                    title += name;
                 }
-                
-                scope.$emit("$changeTitle", title);
```

Comment:
Is this a breaking change since there's no longer an event emitted? Was something relying on this?